extensions/chocolatey-core.extension/extensions/Get-EffectiveProxy.ps1

```diff
@@ -30,29 +30,31 @@ function Get-EffectiveProxy(){
         Write-Verbose "Using `$Env:http_proxy"
         return $env:http_proxy
     }
-
-    # Try chocolatey config file
-    $p = choco.exe config get proxy --limit-output
-    if ($p) {
-        Write-Verbose "Using choco config proxy"
-        return $p
-    }
     
     # Try to get IE proxy
     $key = "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings"
     $r = Get-ItemProperty $key
     if ($r.ProxyEnable -and $r.ProxyServer) {
         Write-Verbose "Using IE proxy settings"
-        return $r.ProxyServer
+        return "http://" + $r.ProxyServer
+    }
+    
+    # Try chocolatey config file
+    $p = 'y' | choco.exe config get proxy --limit-output
```

Comment:
> Mhm... forgot about it. I could look into XML config then instead of calling choco, what do you think ?

that's probably an option, but it's probably frowned upon (don't think it's blacklisted though).

> Your link is something different, already handled by the func as the first method.

Oh right, I just thought perhaps the `$env:chocolateyProxyLocation` would be set to the same value that is returned from `choco config get proxy`.

I think the best bet now though, is to wait on @ferventcoder's thoughts for this first.